### PR TITLE
from branch current -> dev

### PR DESCRIPTION
**Pull request by GulshirinBerdiyeva**
Awaiting approval.

### CODE REVIEW

File: src/main/java/com/epam/task/first/logic/ArrayLogic.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        for (int newMin : array.getElements()){
+            if (min > newMin){
+                min = newMin;
+            }
+        }
+        return min;
+    }
+
+    public void replacementEvenElements(Array array){
+        for (int i = 0; i < array.getElements().size(); i++){
```
Comment (FilippStankevichTR) [inline src/main/java/com/epam/task/first/logic/ArrayLogic.java:30]: one line one dot, so create separate variable for array.getElements()

File: src/main/java/com/epam/task/first/logic/ArrayLogic.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            if (array.getElements().get(i) % 2 == 0){
```
Comment (FilippStankevichTR) [inline src/main/java/com/epam/task/first/logic/ArrayLogic.java:31]: separate variable for element as well

File: src/test/java/com/epam/task/first/logic/ArrayLogicTest.java [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    @Test
+    public void testFindMeanShouldFindMeanWhenNumbersApplied(){
+        //given
+        Array array = new Array(1, 2, 3, 4, 12, 11, 10, 9);
+
+        //when
+        double actual = arrayLogic.findMean(array);
+
+        //then
+        Assert.assertEquals(6.5, actual, 0.);
```
Comment (FilippStankevichTR) [inline src/test/java/com/epam/task/first/logic/ArrayLogicTest.java:89]: usually delta is not 0, something like 0.001

File: target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+D:\work\projects\FirstTask\src\main\java\com\epam\task\first\view\ArrayPrinter.java
```
Comment (FilippStankevichTR) [inline target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst:1]: don't commit anything under target

**Review by FilippStankevichTR — CHANGES_REQUESTED** [review]

File: target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst (hunk 4 above)
Comment (GulshirinBerdiyeva) [reply target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst:1]: Ok, I understood.

**Pull request merged**